cpp/ps0003/cpp_ps0003_solution.cpp

```diff
@@ -2,11 +2,17 @@
 using namespace std;
 int main()
 {
+	// prompting user for number 2/2
 	int x;
-	cout << "enter a namber\n";
+	cout << "enter a number\n";
 	cin >> x;
+
+	// checking if the number is odd or even 3/3
 	if (x % 2 == 0)
 		cout << "the number is even\n";
 	else
 		cout << "the number is odd\n";
-}
\ No newline at end of file
+	
+	return 0;
+}
+// great work 😎 8/10

```

Comment:
- [ ] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code